briskLimbs/briskLimbs
# [feature/BRIS-3] authentication

https://brisklimbs.atlassian.net/browse/BRIS-3

## What it solves
Implements authentication using Laravel and new template

## How this PR fixes it

...

## How to test it
The following checks need to be performed to confirm all works as expected

1. `Login` and `Register` links in navbar point to correct pages
2. Users can `Register`
3. Users can `Login`
4. Password rest link on `Login` page is correct
5. Password reset works as expected

## Tested browsers


## Readiness Checklist

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request
- [x] Pull request title is brief and descriptive (for a changelog entry)

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [x] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`
